docs/UserGuide.md

```diff
@@ -809,11 +812,11 @@ Format: `updateC CONTACT_INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [
 
 Parameters:
 
-* [`NAME`](#name) The name of a contact.
-* [`PHONE_NUMBER`](#phone_number) The phone number of a contact.
-* [`EMAIL`](#email) The email of a contact.
-* [`ADDRESS`](#address) The address of a contact.
-* [`TAG`](#tag) The tag associated with a contact.
+* [`NAME`](#name) The updated name of contact.
+* [`PHONE_NUMBER`](#phone_number) The updated phone number of contact.
+* [`EMAIL`](#email) The updated email of contact.
+* [`ADDRESS`](#address) The updated address of contact.
+* [`TAG`](#tag) The updated tag associated with contact.
```

Comment:
For these lines, I think it's better to write `... of the contact`, to be consistent.

Reply:
done